Bottom: 7169246c1dcdabd9c359682283ebe77b7f4baf40
Top:    35addde75bd544888b32441bf03effff7b0fb979
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:06:21 +0000

Refresh of change-unmodifed-parameters-to

---

diff --git a/src/misc.c b/src/misc.c
index 76b05c0..b1d4304 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -108,7 +108,7 @@ static struct
  * bool variable absolute to false if it leads with a +/- character
  */
 float
-GetValue (char *val, char *units, bool * absolute)
+GetValue (const char *val, const char *units, bool * absolute)
 {
   double value;
   int n = -1;
diff --git a/src/misc.h b/src/misc.h
index bb88a21..82bd194 100644
--- a/src/misc.h
+++ b/src/misc.h
@@ -74,7 +74,7 @@ BoxTypePtr GetArcEnds (ArcTypePtr);
 void ChangeArcAngles (LayerTypePtr, ArcTypePtr, long int, long int);
 char *UniqueElementName (DataTypePtr, char *);
 void AttachForCopy (LocationType, LocationType);
-float GetValue (char *, char *, bool *);
+float GetValue (const char *, const char *, bool *);
 int FileExists (const char *);
 char *Concat (const char *, ...);	/* end with NULL */
